app/controllers/concerns/exception_notifier_custom_data.rb

```diff
@@ -14,7 +14,7 @@ def prepare_exception_notifier
     request.env[
       'exception_notifier.exception_data'
     ] = OpenshiftEnvironment.summary.merge(
-      current_user: current_user.account.account_number
+      current_user: current_user&.account&.account_number
```

Comment:
It is, but I added this as if for whatever reason there's an error in prod before current_user is set, we still get it through the Openshift notifier instead of getting an exception

Reply:
Ok, that's fine.